client/state/login/reducer.js

```diff
@@ -25,6 +33,21 @@ export const isRequesting = createReducer( false, {
 	[ LOGIN_REQUEST_SUCCESS ]: () => false,
 } );
 
+export const redirectTo = createReducer( null, {
```

Comment:
It is stored in different places when we use social login, but I think selector could handle it. Good idea 👍